lib/Support/Debug.cpp

```diff
@@ -41,9 +41,11 @@ namespace glow {
 /// Exported boolean set by -debug-glow option.
 bool DebugFlag = false;
 
+#ifndef NDEBUG
```

Comment:
This is unrelated to the scope of this PR. Please remove.

Reply:
Actually it is related to LLVM 10.0.
The "isCurrentDebugType()" function needs to implemented only if NDEBUG is not defined. 
Otherwise, there is a macro-definition in "include/llvm/Support/Debug.h" which is expanded (when NDEBUG is defined) and will replace "isCurrentDebugType" with "(false)".

#define isCurrentDebugType(X) (false)

This was not the case before LLVM 10.0